tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -467,6 +474,7 @@ else if (col.getColumnInfo().isPrimaryKey() && tableInfo.isPkHandle()) {
         executorBuilder.clear();
         addPushDownFilters();
       } else {
+        dagRequestBuilder.addAllOutputOffsets(outputOffsetsToBeAdded);
```

Comment:
how about we return a dagRequest without offsets and add the offsets afterwards? so we won't forget to add it when new logic is added.

Reply:
done